src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs

```diff
@@ -221,6 +222,11 @@ private void TourFinished(GuidedTourStateEventArgs args)
             }
         }
 
+        private void TourClosed(GuidedTourStateEventArgs args)
+        {
+
+        }
```

Comment:
If we dont do anything here do we need the binding and unbinding?

Reply:
@QilongTang well the OnGuidedTourExited event was just added for setting isGuideExited = true, so we will know that the tour exited, but in the validation of the event I've seen in all the events that are checking the next:
            if (GuidedTourExited != null)
            {
                isGuideExited = true;
                GuidedTourExited(new GuidedTourStateEventArgs(name));
            }
Then if there is no function subscribed to the GuidedTourExited then it won't enter, please let me know if I should remove the validation so we can remove the empty method.

Reply:
Well, I think I am only talking about 
`GuideFlowEvents.GuidedTourExited += TourExited;` and `GuideFlowEvents.GuidedTourExited -= TourExited;` since this function does not do anything, I think we can remove both and this function, no?

Reply:
@QilongTang subscriptions and empty function removed in the next commit:
https://github.com/DynamoDS/Dynamo/pull/13002/commits/6b546b15365e28adfc583a8ff25e79809dfe4956